Week1/index.js

```diff
@@ -0,0 +1,173 @@
+const mysql = require('mysql');
+const fs = require('fs');
+const { promisify } = require('util');
+
+const readFile = promisify(fs.readFile);
+
+// Prepare connection
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+});
+
+const asyncQuery = promisify(connection.query.bind(connection));
+
+// Queries in order, to be executed
+const queries = [
```

Comment:
it's a great choice to use query message structure.
Your create statements are all good.